src/ProtobufInterfaces/proto/point.proto

```diff
@@ -41,6 +41,7 @@ service Point {
     rpc ScheduleTimeoutRight (EnableMovementFailedMessage) returns (google.protobuf.Empty) {}
     rpc ScheduleTimeoutLeft (EnableMovementFailedMessage) returns (google.protobuf.Empty) {}
 
+    rpc ScheduleInitializationTimeout (EnableInitializationFailedMessage) returns (google.protobuf.Empty) {}
```

Comment:
Inconsistent naming: Timeout vs Failed. This is not the only spot, let's use Timeout consistently.